modules/investigating-kernel-crashes.adoc

```diff
@@ -27,36 +27,12 @@ Perform the following steps to enable `kdump` on {op-system}.
 # rpm-ostree kargs --append='crashkernel=256M'
 ----
 
-. By default, the path in which the vmcore will be saved is `/var/crash`. It is also possible to write the dump over the network or to some other location on the local system by editing `/etc/kdump.conf`. For example, assuming `/var/usrlocal/cores` exists, enter the following command to edit `/etc/kdump.conf` to save the vmcore to `/var/usrlocal/cores`:
-+
-[source, terminal]
-----
-# sed -i "s/^path.*/path \/var\/usrlocal\/cores/" /etc/kdump.conf
-----
-+
-For additional information, see `kdump.conf`, a manual page for the `/etc/kdump.conf` configuration file containing the full documentation of available options, and note the comments in `/etc/kdump.conf` and `/etc/sysconfig/kdump`.
+. By default, the path in which the vmcore will be saved is `/var/crash`. It is also possible to write the dump over the network or to some other location on the local system by editing `/etc/kdump.conf`. Note that `kdump` does not support local crash dumps on LUKS-encrypted devices; currently, network dumps must be used if using LUKS. For additional information, see `kdump.conf`, a manual page for the `/etc/kdump.conf` configuration file containing the full documentation of available options, and note the comments in `/etc/kdump.conf` and `/etc/sysconfig/kdump`.
```

Comment:
I think the primary step here is to write the dump, so I would suggest starting with that. I also have a few suggested tweaks for readability and to adhere our docs guidelines, such as using the `[NOTE]` callout. Do you think the following still sticks to your general instructions?

```
. Optional: To write the crash dump over the network or to another location, rather than to the default `/var/crash`path, edit the `/etc/kdump.conf` file. 

[NOTE]
====
Network dumps are required when using LUKS. The `kdump` service does not support local crash dumps on LUKS-encrypted devices.
====

For more information about configuring the `kdump.conf` file, see the manual page for details about `/etc/kdump.conf` and `/etc/sysconfig/kdump`.
```

(Removed "vmcore" since that is no longer referenced with this update.)

Reply:
Thanks, Bob! The `[NOTE]` callout definitely makes things clearer. I think your suggestions improve clarity and I followed them generally, with minor tweaks.

Reply:
Your changes look great, @kelvinfan001 - thank you!